src/CircuitNotation.hs

```diff
@@ -1330,4 +1331,5 @@ defExternalNames = ExternalNames
       Fwd -> GHC.Unqual (OccName.mkTcOcc "Fwd")
       Bwd -> GHC.Unqual (OccName.mkTcOcc "Bwd")
   , trivialBwd = GHC.Unqual (OccName.mkVarOcc "unitBwd")
+  , unsafeConsPat = GHC.Unqual (OccName.mkDataOcc ":>!")
```

Comment:
If it's only an issue on new GHCs, is it reasonable to only use `:>!` for versions where the warning is broken?

Reply:
Yeah, that makes sense, I'll do a fixup